enos/modules/solo_worker/templates/worker.hcl

```diff
@@ -0,0 +1,28 @@
+listener "tcp" {
+  purpose = "proxy"
+  tls_disable = true
+  address = "0.0.0.0"
+}
+
+worker {
+  # Name attr must be unique across workers
+  name = "worker-${id}"
```

Comment:
I might be misunderstanding, but this should always be unique to this worker because `id` comes from the `random_pet` generated in the solo_worker module.

```
    id                   = random_pet.worker.id
```

Reply:
👍 Got it. Misread it. This is fine.